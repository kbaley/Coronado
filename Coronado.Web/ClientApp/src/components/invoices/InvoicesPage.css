.show-template {
  font-size: 20px;
  margin: 10px 0 10px 10px;
  vertical-align: middle;
}