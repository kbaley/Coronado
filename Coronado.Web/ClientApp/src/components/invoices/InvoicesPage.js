import React, { Component } from 'react';
import { connect } from 'react-redux';

export class InvoicesPage extends Component {
  render() {
    return (
      <div>
        <h1>
          Invoices
        </h1>
      </div>
    );
  }
}

function mapStateToProps(state) {
  return {
    invoices: state.invoices,
  }
}

export default connect(
  mapStateToProps,
)(InvoicesPage);
