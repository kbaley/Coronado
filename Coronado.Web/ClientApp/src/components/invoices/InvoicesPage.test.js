import React from 'react';
import { InvoicesPage } from "./InvoicesPage";
import renderer from "react-test-renderer";
import { shallow } from 'enzyme';
// import configureMockStore from 'redux-mock-store';

// const mockStore = configureMockStore();

describe('InvoicesPage tests', () => {
  it('should render the component', () => {
    const tree = renderer
      .create(<InvoicesPage />);
    expect(tree.toJSON()).toMatchSnapshot();
  });

  it('should map invoices to props', () => {
    


  });
});