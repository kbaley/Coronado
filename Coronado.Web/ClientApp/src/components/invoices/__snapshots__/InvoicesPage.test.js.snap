// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InvoicesPage tests should render the component 1`] = `
<div>
  <h1>
    Invoices
  </h1>
</div>
`;
